agent/bench-scripts/postprocess/process-iteration-samples

```diff
@@ -209,9 +216,13 @@ foreach $sample (@json_samples) { # samples 0..N
 						my %sample = (get_label('uid_label') => $sample_hash{$metric_type}{$metric_name}[$i]{get_label('uid_label')});
 						unshift( @{ $iteration{$metric_type}{$metric_name} }, \%sample);
 						$iteration_matching_index = 0;
+						#print "NEW $mapped_uid -- [$metric_type][$metric_name][$i] ==> [$metric_type][$metric_name][$iteration_matching_index]\n";
+					} else {
+						#print "EXISTING $mapped_uid -- [$metric_type][$metric_name][$i] ==> [$metric_type][$metric_name][$iteration_matching_index]\n";
 					}
 					foreach my $key (sort keys %{$sample_hash{$metric_type}{$metric_name}[$i]}) {
 						my $sample_value = $sample_hash{$metric_type}{$metric_name}[$i]{$key};
+						#print "[$metric_type][$metric_name][$i][$key] = $sample_value \n";
```

Comment:
Here and above (ll. 219-221): do we want to keep the debugging output?

Reply:
Removed.